torch/_inductor/scheduler.py

```diff
@@ -691,15 +719,17 @@ class NopKernelSchedulerNode(BaseSchedulerNode):
 
 def debug_triton_code(node: Union["SchedulerNode", "FusedSchedulerNode"]) -> List[str]:
     lines = []
-    is_multi_template = node.is_template() and isinstance(
-        node.get_template_node(), ir.MultiTemplateBuffer
-    )
-    if is_multi_template and node.get_template_node().make_kernel_render is None:
+    multi_template = node.get_template_node()
+    assert multi_template is None or isinstance(multi_template, ir.MultiTemplateBuffer)
+    if multi_template and multi_template.make_kernel_render is None:
         lines.append(f"{node.get_name()} Unfinalized multi template buffer")
     else:
+        from torch._inductor.codegen.triton import TritonScheduling
+
         snodes = (node,) if isinstance(node, SchedulerNode) else node.snodes
         device = snodes[0].get_device()
         backend = node.scheduler.get_backend(device)
+        assert isinstance(backend, TritonScheduling)
```

Comment:
This assert fails. Not sure why CI does not catch it.

Reply:
oh, maybe CI caches it, but this is caught and skipped

Reply:
I ran into this as well.

Reply:
Fixed in #127021